main/index.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+require('./setup-sentry-on-backend')
```

Comment:
```suggestion
require('./setup-sentry')
```

"on backend" is given by the location inside the `main` folder

Reply:
> "on backend" is given by the location inside the main folder

You are right.

I am not a fan of having files with names that also require their path to understand the purpose of the file. 

Having said that, I guess `Sentry.tsx` and `setup-sentry.js` are different enough that I can memorise which one is for the frontend and which is for the backend.

Reply:
I see, depending on the editor this can definitely be a problem! It would be cool if we could find a convention for this 🤔